custom_reports/object_management_api/definitions/goal.yaml

```diff
@@ -1,15 +1,78 @@
 type: object
-allOf:
-  - type: object
+required:
+  - type
+  - attributes
+  - id
+properties:
+  type:
+    type: string
+    description: Type of the object.
+    enum:
+      - Goal
+  id:
+    type: string
+    description: Id of a goal.
+    readOnly: true
+    format: uuid
+    example: dde2dc68-b6bb-45ee-9ac2-868b6181e33f
+  attributes:
+    type: object
+    required:
+      - name
+      - trigger
+      - revenue
+      - website_id
     properties:
-      attributes:
-        type: object
-        required:
-          - website_id
-        properties:
-          website_id:
-            type: string
-            format: uuid
-            description: ID of the website.
-            example: dde2dc68-b6bb-45ee-9ac2-868b6181e33f
-  - $ref: '#/definitions/GoalDetails'
+      website_id:
```

Comment:
don't we want to use refs? We do have         `$ref: '#/components/schemas/WebsiteId'`

Reply:
we have only `$ref: '#/parameters/WebsiteId'` but we could add components section in index.yaml.

Reply:
yes, I was looking at wrong repo.